--- a/webview/assets/app-initial~artifact-tab-content.electron~notebook-preview-panel~app-main~appgen-settings-p~jj50pjos-D3LKdNnF.js
+++ b/webview/assets/app-initial~artifact-tab-content.electron~notebook-preview-panel~app-main~appgen-settings-p~jj50pjos-D3LKdNnF.js
@@ -8630,6 +8630,7 @@
     n = h(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (Wh({
+    enabled: !1,
     beforeSend: de,
     dsn: p,
     environment: Qh,
