--- a/webview/assets/app-initial~app-main~pull-request-code-review~onboarding-page~hotkey-window-thread-page~cha~b76hmflu-CeoeefuW.js
+++ b/webview/assets/app-initial~app-main~pull-request-code-review~onboarding-page~hotkey-window-thread-page~cha~b76hmflu-CeoeefuW.js
@@ -2388,6 +2388,7 @@
     n = V(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (ZNe({
+    enabled: !1,
     beforeSend: Ht,
     dsn: Re,
     environment: iPe,
