--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -80447,6 +80447,7 @@
   let n = PV(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (fD({
+    enabled: !1,
     dsn: LU,
     environment: e.buildFlavor,
     release: RU(n.version),
--- a/.vite/build/window-all-closed-CZr9g6FK.js
+++ b/.vite/build/window-all-closed-CZr9g6FK.js
@@ -40315,6 +40315,7 @@
     i = t.la(e.version);
   return (
     Bz({
+      enabled: !1,
       dsn: t.ca,
       environment: oB.buildFlavor,
       release: i,
