--- a/webview/assets/app-initial~artifact-tab-content.electron~notebook-preview-panel~app-main~first-run~appgen-~f83t1tf1-BBTOXWeK.js
+++ b/webview/assets/app-initial~artifact-tab-content.electron~notebook-preview-panel~app-main~first-run~appgen-~f83t1tf1-BBTOXWeK.js
@@ -1620,7 +1620,7 @@
         })));
   });
 function hn(e) {
-  return `${vn}${_n(e)}`;
+  return _n(e);
 }
 function gn(e) {
   return _n(e);
