--- a/webview/assets/app-initial~app-main~appgen-settings-page~page~appgen-library-page~appgen-page~appgen-setti~ogh9jurw-Ccxu2qV_.js
+++ b/webview/assets/app-initial~app-main~appgen-settings-page~page~appgen-library-page~appgen-page~appgen-setti~ogh9jurw-Ccxu2qV_.js
@@ -3348,7 +3348,7 @@
   return (
     t[1] !== n || t[2] !== r
       ? ((a = (0, YP.jsxs)(Qee, {
-          initialEntries: r,
+          initialEntries: r ?? [window.__ELECTRON_SHIM__.initialRoute],
           unstable_useTransitions: !1,
           children: [i, n],
         })),
