--- a/webview/assets/app-initial~app-main~onboarding-page~hotkey-window-thread-page~quick-chat-window-page~chatg~gvvmoah5-BM6D2haX.js
+++ b/webview/assets/app-initial~app-main~onboarding-page~hotkey-window-thread-page~quick-chat-window-page~chatg~gvvmoah5-BM6D2haX.js
@@ -120437,7 +120437,7 @@
         })));
   });
 function yF(e) {
-  return `${Dot}${Eot(e)}`;
+  return Eot(e);
 }
 function Tot(e) {
   return Eot(e);
