--- a/webview/assets/app-initial~avatarOverlayCompositionSurface~artifact-tab-content.electron~app-main~appgen-s~j5d6n91g-DVu-pwEX.js
+++ b/webview/assets/app-initial~avatarOverlayCompositionSurface~artifact-tab-content.electron~app-main~appgen-s~j5d6n91g-DVu-pwEX.js
@@ -5502,6 +5502,9 @@
       (n[2] = r))
     : (r = n[2]);
   let i = r;
+  window.__ELECTRON_SHIM__.closeSidebar = () => {
+    ht(e, !1, { animate: t });
+  };
   Qe(`toggleSidebar`, i);
   let a;
   (n[3] !== e || n[4] !== t
